src/utils.c

```diff
@@ -135,6 +146,27 @@ void string_strip_delimited(char *str, char a, char b)
         str[iwrite] = 0;
 }
 
+/* see utils.h */
+char **string_to_array(const char *string)
```

Comment:
I guess you implemented `g_strsplit()` here, right?

Reply:
Wow, I'm a dummy, haha. It was right there in front of me! Somehow I didn't see that function, since glib documentation doesn't come up too much when googling.